tools/src/bin/searchfox-tool.rs

```diff
@@ -55,6 +42,10 @@ async fn main() {
                     }
                 }
             }
+        },
+        Err(err) => {
+            println!("Pipeline Error!");
+            println!("{:?}", err);
```

Comment:
Should this be exiting with an error code?

Reply:
Yes, that definitely seems appropriate; I'll have it exit with an error code, thanks!